### PR TITLE
DEV066 add logic at choose place

### PR DESCRIPTION
**Pull request by MariaPaharelava**
Add logic at choose place
[card](https://trello.com/c/qzzE2GTT)

### CODE REVIEW

File: app/views/organizations/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 <%= javascript_pack_tag 'application' %>
+<% p 23456789 %>
```
Comment (Maxxx911) [inline app/views/organizations/show.html.erb]: Delete this

File: db/schema.rb [hunk 2]
```diff
@@ -34,8 +34,8 @@
   end
 
   create_table "images", force: :cascade do |t|
-    t.integer "imageable_id", null: false
-    t.string "imageable_type", null: false
```
Comment (Maxxx911) [inline db/schema.rb]: correct this

File: app/controllers/organizations_controller.rb [hunk 3]
```diff
@@ -20,6 +21,11 @@ def confirm
     redirect_to action: :show, id: @organization.id
   end
 
+  def check_time
+    @organization = Organization.find(params[organization[:organization_id]])
+    render json: Time.new.hour == time_for_group.split('-')[0].split(':')[0].to_i
```
Comment (andreydro) [inline app/controllers/organizations_controller.rb]: Move logic for json to separate method

File: app/models/organization.rb [hunk 4]
```diff
@@ -28,4 +28,11 @@ class Organization < ApplicationRecord
   accepts_nested_attributes_for :events
   accepts_nested_attributes_for :users
   accepts_nested_attributes_for :organization_events
+  enum group: [0, 1, 2]
+
+  def create_group
+    organizations_group = Organization.all.map(&:group)
```
Comment (andreydro) [inline app/models/organization.rb]: Use pluck here

File: app/models/organization.rb [hunk 5]
```diff
@@ -28,4 +28,11 @@ class Organization < ApplicationRecord
...
+    count_of_each_group = [0, 1, 2].map { |item| organizations_group.count(item) }
```
Comment (andreydro) [inline app/models/organization.rb]: Move array to constant

File: app/views/organizations/show.html.erb [hunk 6]
```diff
@@ -1,4 +1,7 @@
 <%= javascript_pack_tag 'application' %>
+<% if flash[:message] &&  current_user.organization_ids.index(@organization[:id]) %>
```
Comment (andreydro) [inline app/views/organizations/show.html.erb]: Move check to helper method

**Pull request merged**